core/src/main/resources/addonlangs.json

```diff
@@ -23,16 +23,22 @@
     "baseUri":"http://www.wikidata.org",
     "apiUri":"https://www.wikidata.org/w/api.php"
   },
-  "core": {
-    "wikiCode": "core",
-    "name": "Core Directory",
-    "isoCode": "en",
-    "iso639_3": "eng"
-  },
   "commons": {
     "wikiCode": "commons",
     "name": "Commons",
     "isoCode": "en",
+    "iso639_3": "eng",
+    "dbpediaDomain":"commons.dbpedia.org",
```

Comment:
We identified [this commit](https://github.com/dbpedia/extraction-framework/commit/e0e82f90acf147687412ef79a53f6b5bc4859b02#diff-25daade9308e37621b81033b0a9f4860L93) as the reason for the issue we ran into as the special commons handling was removed.